app/middleware/cache_headers.py

```diff
@@ -26,9 +26,8 @@ def __call__(self, request):
 
         # Code to be executed for each request/response after
         # the view is called.
-
-        if request.path in ['/checker', '/get-token']:
-            # never cache the /checker and /get-token endpoints.
+        if request.path.split('/')[-1] in ['checker', 'get-token', settings.HEALTHCHECK_ENDPOINT]:
```

Comment:
To avoid forgetting this check if one of those path would change in the future, I would use the exact same string as in url.py, this way they can be found with a search and replace; `['checker', f'{API_BASE}/get-tocken', f'{API_BASE}/{HEALTH_CHECK_ENDPPOINT}']`